src/Covid19Dashboard/Covid19Dashboard.less

```diff
@@ -152,3 +152,47 @@
 .overlays .spinner__svg {
   fill: #fff;
 }
+
+
+//Styling for Gen3 Dashboard
+
+.gen3-dashboard_container {
+  width: 100%;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+
+.gen3-dashboard_row {
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
+  // padding: 0.5vh;
+}
+
+.gen3-dashboard_col {
```

Comment:
i'm not sure what is causing this, but when i resize the window:

at first the charts scale when i make the window wider:
![Screen Shot 2022-12-16 at 1 19 58 PM](https://user-images.githubusercontent.com/4224001/208173031-c26c0816-df1e-4ee7-9407-c6a456d429b3.png)

but after a certain width, the chart on the left gets very small:
![Screen Shot 2022-12-16 at 1 20 10 PM](https://user-images.githubusercontent.com/4224001/208173038-866b125b-b51c-42a8-a682-6718eee8169b.png)

Reply:
This is a really good find. I fixed it now.